spec/controllers/nodes_controller_spec.rb

```diff
@@ -134,6 +134,7 @@
           { hostname: "minion0.example.com" },
           { hostname: "minion1.example.com" }
         ]
+        allow(Velum::Salt).to receive(:orchestrate)
```

Comment:
Here we are allowing message, but we are not using its counterpart `expect(Velum::Salt).to receive(:orchestrate)`, right? Here we are basically setting the double, but not using it.